components/wpt-results.html

```diff
@@ -46,59 +120,170 @@
 
       static get properties() {
         return {
+          query: {
+            type: String,
+            value: '',
+            observer: '_queryChanged'
+          },
           specs: {
             type: Array,
-            observer: '_specsChanged',
             value: []
-          }
+          },
+          testRuns: {
+            type: Array,
+          },
         };
       }
 
-      connectedCallback() {
+      async connectedCallback() {
         super.connectedCallback();
 
-        this._fetchResults()
+        const testFileResults = await Promise.all(this.testRuns.map(testRun => {
+          return this._fetchResults(testRun.results_url)
+        }))
+
+        let specs = []
+
+        testFileResults.forEach(result => {
+          const testFiles = result.testFiles
+          const resultsURL = result.resultsURL
```

Comment:
Ah, interesting! Yep, there's no reason why we have to use semicolons. The most important thing to me is consistency.

(Speaking of which, have you been using eslint and golint for this repo? If you are, feel free to add any relevant config/rc files or notes about their use in a top-level README.md file.)